packages/client/src/Conversation.tsx

```diff
@@ -269,7 +269,7 @@ function Presentable({
         }
       />
       <Collapse in={expanded} timeout="auto">
-        <CardContent id="message-card-content">
+        <CardContent id={`${presentable.id}-card-content`}>
```

Comment:
In cases like this where it is important that two strings are identical it is useful to define a variable inside of writing the string twice. A variable can make your intention more clear to developers who come along later who might otherwise change one string and not know that the other string should be changed to match. Just something to keep in mind in the future.